src/cpu/aarch64/brgemm/jit_brgemm_kernel.cpp

```diff
@@ -1470,8 +1500,10 @@ void jit_brgemm_kernel_t::gemm_microkernel_sve512(int bd_block2,
                     const auto bd_by_load_bytes
                             = (bd >= bd_e - rows_by_load_bytes
                                     || brg.brgattr.wary_A_k_tail_read);
-                    broadcast(bcst(), A_offset(bd, rd),
-                            have_to_load_bytes && bd_by_load_bytes, brg.dt_a);
+                    int should_broadcast = static_cast<int>(
```

Comment:
What does this mean? We have a variable called `should_broadcast` but then we call `broadcast` even if it is not true, are they two different kinds of broadcasting? If so it would be great to make that clear in the variable/function names